global_routes_test.go

```diff
@@ -364,3 +364,43 @@ func TestMongoSlowQueries(t *testing.T) {
 		assert.Equal(test.is_collscan, fields["is_collscan"])
 	}
 }
+
+func TestRDSSlowQueries(t *testing.T) {
+	t.Log("Base case: doesn't route empty log")
```

Comment:
can you make these into tabulated tests? if you right click on `rdsSlowQueries` in VSCode there should be a menu option like `GO: Generate Unit Tests for Function`